pkg/resource/training_job/hooks.go

```diff
@@ -65,3 +75,21 @@ func (rm *resourceManager) customSetOutput(r *resource) {
 
 	svccommon.SetSyncedCondition(r, trainingJobStatus, &resourceName, &trainingJobModifyingStatuses)
 }
+
+// Requeue whenever Warmpool cluster is in Available or Inuse state.
+func (rm *resourceManager) customSetWarmPoolOutput(r *resource) error {
```

Comment:
can we reuse the customSetOutput method above by adding another return parameter? it would reduce the code being maintained

Reply:
done